doc/themes/scikit-learn/static/nature.css_t

```diff
@@ -1383,3 +1383,41 @@ li#other-versions {
   }
 
 }
+
+
+span.copybutton {
+    font-size: 13px !important;
+    font-weight: bold !important;
+    font-family: monospace;
+    border-color: transparent !important;
+    color: rgb(119,119,119) !important;
+    top: 20px !important;
+    padding: 0px 4px 2px !important;
+    background: transparent no-repeat none !important;
+}
```

Comment:
This is not included in the extension?

Reply:
The naming is confusing here. The `>>>` prompt button is named as copybutton. I just try to re-arrange `Hide prompts and output` button.  This is not the one I am adding.